src/prm.a51

```diff
@@ -44,38 +44,66 @@ $INCLUDE (83c552.mcu)
 ;----------------------------------------
 ; SFRs
 ;----------------------------------------
-AUXR1		EQU	0a2h
+AUXR1				EQU	0a2h
 
-RAM		EQU	030h
-RAMbit		EQU	020h
+RAM					EQU	030h
+
+RAMbit				EQU	020h
+					BSEG AT 0h
 ;----------------------------------------
 ; Variables
 ;----------------------------------------
-serial_latch_lo	EQU	RAMbit+0	; Valeur du premier verrou
-serial_latch_hi	EQU	RAMbit+1	; Valeur du deuxieme verrou
+slatch_lo:			DBIT 8			; Valeur du premier verrou
+slatch_hi:			DBIT 8			; Valeur du deuxieme verrou
```

Comment:
Yes, thats correct. 
My goal was to adress bitvariables via a symbolic name, which is much better than using a "byte" name + bit number.
(It's hard to understand the code in this case).
But using logical names for bits means you have to define all previous bits, thats what I did.
I know, that lots of variables are not used, yet, but If we modify the code later we could use this bit names instead of byte names and bit number. 
So if you agree I would stay with current definitions.

Reply:
You can't leave the memory assigned and unused. If you do it, change also the code to remove the previous usage. The bit access memory is only16 byte long.

The second point is : if you use direct bit assignement, you loose the mapping with the byte access area ? And this is used for some of them

Reply:
may be this is a missunderstanding or I do not get the point.
The momory is not unused. The **memory** definition I did is reflecting exactly the same layout you did with the equations, means the same memory is used as before.

My goal was to have the same layout and move from equations to real bit definitions step by step.

I also took care, that the alignment for the byte access area is not effected. I even testet that my definition resulted to the same adresses as they were before. (when I assemble the code, I don't delete the *.lst file, I did the checks by controlling the adresses manually).

If you don't like the bit definitions at all I also could reverse the changes. 
If you are fine with the bit definitions, I can do the remaining bit definitions and migrate the code from modifying 
bits directly (name.0, name.1...) to named bits (which newcomers like me can understand much better).

Sorry for generating all this effort to you, 
We should have discussed my plan to change the layout before doing it. 
(my assumption was you plan to get rid off this assembly code anyway, so my changes do not affect your work at all)

Reply:
as already stated, changes done as requested and discussed yesterday